boards/arm/raytac_mdbt50q_db_33_nrf52833/doc/index.rst

```diff
@@ -0,0 +1,210 @@
+for a complete list of Raytac MDBT50Q-DB-33 board hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.13
+* LED2 (red) = P0.14
+* LED3 (blue) = P0.15
+
+
+Push buttons
+------------
+
+* BUTTON1 = SW1 = P0.11
+* BUTTON2 = SW2 = P0.12
+* BUTTON3 = SW3 = P0.24
+* BUTTON4 = SW4 = P0.25
+
+UART
+----
+* RXD = P0.08
+* TXD = P0.06
+* RTS = P0.05
+* CTS = P0.07
+
+Programming and Debugging
+*************************
+
+Applications for the ``raytac_mdbt50q_db_33_nrf52833`` board configuration can be
+built, flashed, and debugged in the usual way. See
+:ref:`build_an_application` and :ref:`application_run` for more details on
+building and running.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the board nRF52840 DK
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: raytac_mdbt50q_db_33_nrf52833
+   :goals: build flash
+   
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic boards with a
+Segger IC.
+
+
+Testing the LEDs and buttons in the MDBT50Q-DB-33
+*************************************************
+
+There are 2 samples that allow you to test that the buttons (switches) and LEDs on
+the board are working properly with Zephyr:
+
+.. code-block:: console
+
+   samples/basic/blinky
+   samples/basic/button
+
+You can build and flash the examples to make sure Zephyr is running correctly on
+your board. The button and LED definitions can be found in
+:zephyr_file:`boards/arm/raytac_mdbt50q_db_33_nrf52833/raytac_mdbt50q_db_33_nrf52833.dts`.
+
+Selecting the pins
+==================
+
+Pins can be configured in the board pinctrl file. To see the available mappings,
+open the `MDBT50Q-DB-33 Specification`_, chapter 2.5 'Pin Assignment'.
+Select the pins marked 'General-purpose I/O'. Note that pins marked as 'low-frequency I/O 
+only' can only be used in under-10KHz applications. They are not suitable for SPI, I2C, 
+UART, and PWM.
+
+References
+**********
+
+.. target-notes::
+
+.. _MDBT50Q-DB-33 website:
+	https://www.raytac.com/product/ins.php?index_id=81
+.. _MDBT50Q-DB-33 Specification:
+	https://www.raytac.com/download/index.php?index_id=43
+.. _MDBT50Q-DB-33 Schematic:
+	https://www.raytac.com/upload/catalog_b/134ade06b5db3dd5803d27c5b17f22f3.jpg
```

Comment:
One of these commits is wrong because both have the same links but with different files

Reply:
Sorry, I copied MDBT50Q-DB-40 but forgot to change the link.